### DISCUSSION TITLE
van-field组件在设置type="textarea"的情况下，如何自动聚焦，键盘自动弹起

### DISCUSSION THREAD

**Discussion by hmyjyghh** [youzan/vant#11920] [Q&A] [answered]
<van-cell-group inset>
          <van-field
            ref="inputRef"
            v-model="message"
            rows="3"
            label=""
            type="textarea"
            maxlength="200"
            placeholder="分享你的见解"
            :autofocus="true"
            show-word-limit
          />
        </van-cell-group>

**Comment by inottn** [comment]
可以获取组件实例，调用 focus 方法聚焦。

**Comment by wjw-gavin** [comment, marked as answer]
https://github.com/youzan/vant/issues/11737  参考下这个

**Reply by hmyjyghh** [reply]
好的，
![3A1D4200](https://github.com/youzan/vant/assets/22025175/7546a9f0-c833-43b3-bbe5-ee741c86e817)
